scripts/make_release_branch.py

```diff
@@ -1,15 +1,62 @@
-from common import get_patch_version, inc_patch_version, GRADLE_PROPERTIES
+import argparse
+import datetime
+import json
+import re
+from urllib.request import urlopen
+
+import github_connect
+from common import get_patch_version, inc_patch_version, GRADLE_PROPERTIES, env
 from git import git_command
+from make_next_milestone import RELEASE_MANAGER_RE
+
+
+def send_slack_message():
+    slack_ids = json.loads(args.slack_ids)
+
+    milestone = github_connect.get_current_milestone(repo=env("GITHUB_REPOSITORY"))
+
+    release_version = milestone["title"]
+    release_manager = re.search(RELEASE_MANAGER_RE, milestone['description']).group(1)
+    link = milestone['html_url']
+    date = datetime.datetime.strptime(milestone["due_on"], "%Y-%m-%dT%H:%M:%SZ").date()
+
+    slack_user = slack_ids.get(release_manager)
```

Comment:
It would be nice to fail here if `slack_user` is not found and white something useful in error message

Reply:
It might not be found if we forgot to update the secrets.
I think, in this case it is better to get a message about the release branch, but with a note that the user was not found. Otherwise, we may simply not notice the failure. What do you think? 

And I'll add a comment with a reminder to update secrets with usernames in the milestone creation file.

Reply:
The investigation showed that if you pass an incorrect USERID, Slack will not send a message and will return a 400 code. :(
I see two options:
pass USERID of someone who is definitely not a release manager (mine, for example). 
Fail with an error, but I don’t understand how we will notice it in time.